daycan-front/apps/admin/src/pages/member/components/MemberDetailCard/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * MemberDetailCard이 컴포넌트는 수급자 상세 카드 컴포넌트입니다.
+ * DetailCardLayout 컴포넌트를 사용하여 수급자 상세 카드 레이아웃을 구성합니다.
+ * @author 소보길
+ */
+
+export { MemberDetailCard } from "./MemberDetailCard";
```

Comment:
Detail
```suggestion
/*
 * MemberDetailContent이 컴포넌트는 수급자 상세 콘텐츠 컴포넌트입니다.
 * DetailCardLayout 컴포넌트 내에서, 사용되며, 수급자의 상세한 내용을 담는 컴포넌트입니다.
 * @author 소보길
 */

export { MemberDetailContent } from "./MemberDetailContent";
```

동일하게 명명법 수정하면 좋을 것 같아요! Card-> Content